### PR TITLE
Also expect the iface dynamic flag on the 10th pos

### PR DESCRIPTION
**Pull request by markVnl**
NethServer/dev#6422

### CODE REVIEW

File: root/etc/e-smith/events/actions/nethserver-base-initialize-db [hunk 1]
```diff
@@ -96,7 +96,7 @@ sub fetch_ifcfg_props
         my($id, $dev, $family, $cidr, $brd_family, $brd_address, $scope_lbl, $scope, $dynamic, $end) = split(/\s+/, $_, 10);
         my $dhclient_running = (system("ps -A -o cmd | grep -E '(/| )dhclient .' | grep -q $ifname") == 0);
 
-        if ($dhclient_running || $dynamic =~ 'dynamic') {
+        if ($dhclient_running || $dynamic =~ 'dynamic' || $end =~ 'dynamic') {
```
Comment (markVnl) [inline root/etc/e-smith/events/actions/nethserver-base-initialize-db:99]: If I understand correctly  `=~ `  in  `$end =~ 'dynamic'` means it searches for the pattern `dynamic` in `$end`

File: root/etc/e-smith/events/actions/nethserver-base-initialize-db [hunk 2]
```diff
@@ -96,7 +96,7 @@ sub fetch_ifcfg_props
         my($id, $dev, $family, $cidr, $brd_family, $brd_address, $scope_lbl, $scope, $dynamic, $end) = split(/\s+/, $_, 10);
```
[the remark this answers is not in the document]
Comment (gsanchietti) [reply root/etc/e-smith/events/actions/nethserver-base-initialize-db:96]: The current one is good.

File: root/etc/e-smith/events/actions/nethserver-base-initialize-db (hunk 1 above)
Comment (gsanchietti) [reply root/etc/e-smith/events/actions/nethserver-base-initialize-db:99]: Exactly :)

**Review by gsanchietti — APPROVED** [review]
It seems to work without regressions, thank you!

It's now time for an extensive testing ;)

**Pull request merged**